### DISCUSSION TITLE
Starter assignment button does not function

### DISCUSSION THREAD

**Discussion by brantfaircloth** [community/community#136580] [GitHub Education]
### Hi everyone,

When attempting to create the Github starter assignment, it appears that the "Create assignment" button does nothing.  I can't click it (attempted using Safari and Firefox).  This is in a new classroom that I named testing.  I tried to do the same in an actual classroom and the same behavior occurred.

<img width="1535" alt="Screenshot 2024-08-22 at 12 13 24" src="https://github.com/user-attachments/assets/8abfe352-df5d-41a8-b8d9-7de251308f3c">

**Comment by misterdorm** [comment]
I'm having the same experience.  I'll paste the error from the Chrome dev console below.

@brantfaircloth did you ever get this working?

<img width="522" height="121" alt="Screenshot 2025-07-15 at 4 05 36 PM" src="https://github.com/user-attachments/assets/8e5651b3-a5ed-4a95-8fbb-47348d64eab6" />

**Reply by brantfaircloth** [reply]
I did not. Nor did i hear anything from @github.

**Reply by misterdorm** [reply]
Thanks for the response.  I can't even find the original repo.  So I guess I'm going to go a different direction with this.  Thank you!
